### ISSUE TITLE
Training on custom dataset

### ISSUE THREAD

**Issue by rahulmoorthy19** [microsoft/FocalNet#10]
Hi Jianwei,

Thank You for providing the official implementation of Focalnet. I faced issues while trying to use the model for my research. I am working on a binary classification model with an input size of 360x880 and I have made the related adjustments for the same. I am using the Focalnet tiny lrf model specifically. It was observed that the training BCEwithlogits loss was not decreasing at all and stayed around 0.69 the whole time of 30 epochs while the accuracy stayed around 50%. For better performance, I also tried finetuning the Imagenet model while ignoring the head weight which resulted in the same outcome. Just wanted to understand, if you might have any idea about what might cause this issue.

**Comment by jaideep11061982** [comment]
@rahulmoorthy19 how did you solve the problem ,how to train and infer
